e2e/e2etests/test_migrate_chain_support.go

```diff
@@ -31,7 +31,7 @@ const EVM2RPCURL = "http://eth2:8545"
 
 // EVM2ChainID is the chain ID for the additional EVM localnet
 // We set Sepolia testnet although the value is not important, only used to differentiate
-var EVM2ChainID = chains.SepoliaChain.ChainId
+var EVM2ChainID = chains.Sepolia.ChainId
```

Comment:
Consider renaming this var to `EVMSeploliaChainID`

Reply:
[b3ce9fd](https://github.com/zeta-chain/node/pull/2258/commits/b3ce9fda74f8206eb780436e272d23bd912ef7e5)